src/components/CategoryDropdown.module.scss

```diff
@@ -1,38 +1,38 @@
 .categoryDropdown {
   position: relative;
-  top: -59px;
-  min-width: 11rem;
+  min-width: 13rem;
 }
 
 .menuButton {
   width: 100%;
   margin: 0;
-  padding: 1rem;
+  padding: 1rem 1rem 1rem 3rem;
   font-size: 1rem;
-  background: #fff;
-  border: 1px solid var(--link-decoration-color);
-  border-top-left-radius: 5px;
-  border-top-right-radius: 5px;
+  background: #fff url(/ui/arrow-down-s-line.svg) 1rem 50% no-repeat;
+  color: var(--link-color);
+  background-size: 20px;
   text-align: left;
-  text-transform: capitalize;
```

Comment:
I don't think this line should have been removed. We end up with the selected item saying "blood" instead of "Blood"